test/integration/sanTokenCirculationTest.js

```diff
@@ -1,6 +1,7 @@
 const { testFieldTypes } = require('../helper.js')
-const { testHandlesNullData } = require('../integration_helper.js')
-const { slug, from, to } = require('../setup.js')
+const { testHandlesNullData, testTimeBound } = require('../integration_helper.js')
+const { slug, from, to, days, numberOfDays, formatDate } = require('../setup.js')
+const currency = ''
 
 describe('SAN_TOKEN_CIRCULATION', () => {
```

Comment:
The old implementation does not have a timebound argument. Add a test here that validates it will work properly w/o it.

Reply:
Similar to MVRV comment, there should be test for old and new implementation.